setup.cfg

```diff
@@ -5,7 +5,7 @@ universal=0
 
 [versioneer]
 VCS = git
-style = pep440
```

Comment:
Do you have a link or quick summary explaining this change?

Reply:
oh oops that was a left-over from debugging. I should clean it up